src/ansys/fluent/core/session.py

```diff
@@ -280,42 +276,30 @@ def force_exit_container(self) -> None:
         self.fluent_connection.force_exit_container()
 
     def upload(self, file_name: str, remote_file_name: Optional[str] = None):
-        """Upload a file to the server supported by `PyPIM<https://pypim.docs.pyansys.com/version/stable/>`.
+        """Upload a file to the server.
 
         Parameters
         ----------
         file_name : str
             Name of the local file to upload to the server.
         remote_file_name : str, optional
             Name of the file to be created on the server.
-            If omitted, will maintain the file name from parameter ``file_name``.
-            Directory specification is not supported.
+            If a filename is omitted, the filename from the ``file_name`` parameter
+            is used. Directory specification is not supported.
```

Comment:
```suggestion
            Defaults to ``file_name`` if unspecified. Directory specification is not supported.
```
N.b. all the described behaviour is currently the responsibility of the abstract file transfer service. If we want to state this as the actual behaviour regardless of what file transfer service is in use, maybe we should actually enforce it here (set remote_file_name = file_name if remote_file_name is None).

Reply:
@seanpearsonuk I think we should remove `remote_file_name` because it is particular to `PIMFileTransferService` only.

Reply:
@raph-luc @seanpearsonuk 

`Directory specification is not supported.` is not required in this context (upload file) because by default PIM does not support it. We will remove this now.